src/main/java/frc/robot/RobotContainer.java

```diff
@@ -137,6 +147,20 @@ public RobotContainer() {
 
     autoChooser = new LoggedDashboardChooser<>("Auto Choices", AutoBuilder.buildAutoChooser());
 
+    // Default command for Elevator
+    elevator.setDefaultCommand(
+        elevator.runManualCommand(
+            () -> MathUtil.applyDeadband(coDriverController.getLeftY(), 0.1)));
+
+    // Lock to 0° when A button is held
+    controller
```

Comment:
This should be in configure button bindings (even though we'll only use this for debug)